frontend/src/app/api/auth/google/route.ts

```diff
@@ -0,0 +1,26 @@
+import { NextRequest, NextResponse } from "next/server";
+import { setTokenCookie } from "@/lib/authApi";
+
+export async function POST(req: NextRequest) {
+  const { state } = await req.json();
+
+  const response = await fetch(
```

Comment:
Shouldnt response be wrapped up in try-catch block? What if hitting the domain that does not exist?

Reply:
You are right, i am changing that now

Reply:
And maybe handling also 400/401/403/404?

Reply:
Message is going from the backend, i don't define it, i don't think this is necessary